pkg/dispatcher/config.go

```diff
@@ -94,6 +97,9 @@ func (config *Config) DetermineClusterForJob(jobBase prowconfig.JobBase, path st
 	if isSSHBastionJob(jobBase) {
 		return config.SSHBastion, false
 	}
+	if _, ok := jobBase.Labels[api.KVMDeviceLabel]; ok && len(config.KVM) > 0 {
+		return config.KVM[len(jobBase.Name)%len(config.KVM)], false
```

Comment:
Huh I don't get it. We index by len(job name)? Is this supposed to distrubute eligible jobs between kvm-enabled clusters based on job name length as a semi-random key? 

If so, can this get a comment?

Reply:
A comment added.